docs/developer/actors/publish.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: "Publish"
+date: 2023-12-29T11:00:00+00:00
+sidebar_position: 6
+draft: false
+---
+
+Once you're finished iterating on your actor component, you can publish it to any OCI compliant registry that supports artifacts. Throughout our examples and wasmCloud official artifacts we use [Docker Hub](https://hub.docker.com), AzureCR, and GitHub CR, but other registries should work as well. For local development and testing of the flow, refer to the [Running a local OCI registry](/docs/developer/workflow#running-a-local-oci-registry) section to run a local registry in a Docker container.
+
+Once you have a registry available, you can use the [wash push](/docs/cli/push) and [wash pull](/docs/cli/pull) commands to push and pull actor components from OCI registries. For example, to pull a wasmCloud example locally:
+
+```shell
+wash pull wasmcloud.azurecr.io/hello:0.1.7
+```
+
+If you're working with an unauthenticated registry, you can directly push artifacts to it by specifying an image reference and the local file:
+
+```shell
+wash push localhost:5000/v2/wasmcloud/hello:0.1.7 ./hello.wasm
+```
+
+## Authenticated Registries
+If you're working with an OCI registry that requires credentials, `wash` supports basic authentication via the `--username` and `--password` flags. Alternatively, you may opt to export your credentials as environment variables to avoid exposing them on the command line. This method of authentication works for both pushing and pulling.
+
+```shell
+export WASH_REG_USER=wasmcloud
+export WASH_REG_PASSWORD=supersecret
+wash push wasmcloud.azurecr.io/hello:0.1.7 ./hello.wasm
+```
+
+If your wasmCloud host is pulling from a registry that requires credentials, you can specify them when starting the host:
+```shell
+export OCI_REGISTRY=wasmcloud.azurecr.io
```

Comment:
Nit: I wouldn't use wasmcloud.azurecr.io as the example here, since it doesn't require credentials to pull artifacts